### PR TITLE
pythonPackages.cairosvg: 2.5.1 -> 2.5.2; pythonPackages.celery: add missing requirement; cleanups, format,

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rmcgibbo** [comment]
Do you understand why cairosvg and celery don't show up as changed attrs according to the `^.^!` ofborg eval? https://gist.github.com/GrahamcOfBorg/2561fe0bc34198d15cb9585da86659e0

**Comment by rmcgibbo** [comment]
Answering my own question -- they must be python2-only and we don't recurse into those attrsets. At least that's my guess.

**Comment by SuperSandro2000** [comment]
> Answering my own question -- they must be python2-only and we don't recurse into those attrsets. At least that's my guess.

I should have tested everything in python 3. Maybe my title mislead it?

**Comment by rmcgibbo** [comment]
I don't know -- I don't think it's your fault, I just don't understand why ofborg doesn't see that `python38Packages.celery` has changed.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 115361` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
3 packages marked as broken and skipped:
</summary>
<ul>
<li>csvkit</li>
<li>python38Packages.agate-sql</li>
<li>python39Packages.agate-sql</li>
</ul>
</details>
<details>

  <summary>1 package failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>ansible-lint: log was empty</li>
  </ul>
</details>
<details>

  <summary>2 packages failed to build and are new build failures:</summary>
  <ul>
<li>(python38Packages.ansible-lint): log was empty</li>
<li>biboumi: log was empty</li>
  </ul>
</details>
<details>
<summary>
12 packages built:
</summary>
<ul>
<li>ansible (ansible_2_10)</li>
<li>ansible_2_8</li>
<li>ansible_2_9 (python38Packages.ansible)</li>
<li>kargo</li>
<li>palp</li>
<li>python38Packages.ansible-kernel</li>
<li>python38Packages.ansible-runner</li>
<li>python38Packages.dopy</li>
<li>python38Packages.pytest-ansible</li>
<li>python39Packages.ansible</li>
<li>python39Packages.dopy</li>
<li>python39Packages.pytest-ansible</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 115361` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
3 packages marked as broken and skipped:
</summary>
<ul>
<li>csvkit</li>
<li>python38Packages.agate-sql</li>
<li>python39Packages.agate-sql</li>
</ul>
</details>
<details>

  <summary>1 package failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>ansible-lint: log was empty</li>
  </ul>
</details>
<details>

  <summary>1 package failed to build and are new build failure:</summary>
  <ul>
<li>(python38Packages.ansible-lint): log was empty</li>
  </ul>
</details>
<details>
<summary>
13 packages built:
</summary>
<ul>
<li>ansible (ansible_2_10)</li>
<li>ansible_2_8</li>
<li>ansible_2_9 (python38Packages.ansible)</li>
<li>biboumi</li>
<li>kargo</li>
<li>palp</li>
<li>python38Packages.ansible-kernel</li>
<li>python38Packages.ansible-runner</li>
<li>python38Packages.dopy</li>
<li>python38Packages.pytest-ansible</li>
<li>python39Packages.ansible</li>
<li>python39Packages.dopy</li>
<li>python39Packages.pytest-ansible</li>
</ul>
</details>

File: pkgs/tools/text/csvkit/default.nix [hunk 1]
```diff
@@ -23,17 +23,22 @@ python3.pkgs.buildPythonApplication rec {
...
   ];
 
+  pythonImportsCheck = [ "csvkit" ];
+
   meta = with lib; {
     description = "A suite of command-line tools for converting to and working with CSV";
     maintainers = with maintainers; [ vrthra ];
     license = licenses.mit;
     homepage = "https://github.com/wireservice/csvkit";
+    # FAIL: test_to_sql_create_statement_with_schema (tests.test_agatesql.TestSQL)
```
Comment (dotlambda) [inline pkgs/tools/text/csvkit/default.nix:41]: This has nothing to do with csvkit!

File: pkgs/development/python-modules/cairosvg/default.nix [hunk 2]
```diff
@@ -1,25 +1,44 @@
...
+  version = "2.5.2";
   disabled = !isPy3k;
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "bfa0deea7fa0b9b2f29e41b747a915c249dbca731a4667c2917e47ff96e773e0";
+    sha256 = "sha256-sLmSnPXboAUXjXRqgDb88AJVUPSYylTbYYczIjhHg7w=";
   };
 
+  buildInputs = [ pytest-runner ];
```
Comment (FRidh) [inline pkgs/development/python-modules/cairosvg/default.nix:26]: nativeBuildInputs because it is required by the build system

**Comment by dotlambda** [comment]
@SuperSandro2000 It would be nice if you stopped merging your own premature pull requests: https://github.com/NixOS/nixpkgs/pull/115388

File: pkgs/tools/text/csvkit/default.nix [hunk 3]
```diff
@@ -23,17 +23,22 @@ python3.pkgs.buildPythonApplication rec {
...
+    setuptools
   ];
 
   checkInputs = with python3.pkgs; [
-    nose pytestCheckHook
+    nose
+    pytestCheckHook
   ];
 
+  pythonImportsCheck = [ "csvkit" ];
```
Comment (dotlambda) [inline pkgs/tools/text/csvkit/default.nix:34]: It feels weird to use `pythonImportsCheck` for an application.

File: pkgs/tools/text/csvkit/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/tools/text/csvkit/default.nix:41]: Oh, you are right. sorry.

**Pull request merged**